static/js-src/cs-client.js

```diff
@@ -239,6 +243,16 @@ class ChromeStatusClient {
     return this.doGet(`/features?q=${userQuery}`);
   }
 
+  // Fielddefs API
+  getFieldDefs() {
+    return this.doGet(`/fielddefs`);
+  }
+
+  // Processes API
+  getProcess(featureId) {
```

Comment:
Like with the URLs, if the name is `getProcess`, I would expect the argument to be a feature_type because that is how we identify processes.  It's fine to use `featureId`, but in that case I think a better name would be `getProcessFor(featureId)` or `getProcesByFeature(featureId)`, or `getFeatureProcess(featureId)`.

Reply:
Changed it to `getFeatureProcess(featureId)`